ReactNativeClient/lib/components/screens/note.js

```diff
@@ -329,9 +342,9 @@ class NoteScreenComponent extends BaseScreenComponent {
 
 		shared.uninstallResourceHandling(this.refreshResource);
 
-		// if (Platform.OS !== 'ios' && this.state.fromShare) {
-		// 	ShareExtension.close();
-		// }
+		if (Platform.OS === 'android' && this.state.fromShare) {
```

Comment:
You've nicely abstracted away the platforms in lib/share.js you you can remove the platform check here.

Reply:
Done